data/api-impl/build.gradle

```diff
@@ -50,6 +50,13 @@ kotlin {
             }
         }
         iOSMain {
+            final String mode = project.findProperty("XCODE_CONFIGURATION")?.toUpperCase() ?: 'DEBUG'
```

Comment:
`projectsList` in ioscombined can handle this, so I guess this diff is not required. Didn't it work without this chunk? 👀

Reply:
Yes. It works without this.